### DISCUSSION TITLE
.venv/Scripts/activate: No such file or directory

### DISCUSSION THREAD

**Discussion by jaspreet-dimension** [AcademySoftwareFoundation/OpenRV#609] [Build Issues] [answered]
This is probably an oversight somewhere but when building on Windows via MSYS2 MINGW64 and I'm getting this error when running rvbootstrap

```
rvbootstrap
-bash: .venv/Scripts/activate: No such file or directory
```

Looks like it is coming from rvenv_shell

```
$ type rvenv_shell
rvenv_shell is a function
rvenv_shell ()
{
    local activate_path=".venv/bin/activate";
    if [[ "$OSTYPE" == "msys"* ]]; then
        activate_path=".venv/Scripts/activate";

```

There's no activate file inside my Python .venv/Scripts folder. Did I miss something?

**Comment by cedrik-fuoco-adsk** [comment]
Hello @jaspreet-dimension,

The issue you're experiencing is known to occur when using Python from the msys2 package. The path to the activate script is different with msys2's Python package. 

Our **recommendation** is to install Python using the official installer and update the PATH environment variable accordingly.

For detailed instructions, please refer to our documentation:

I**nstalling Python 3:** https://aswf-openrv.readthedocs.io/en/latest/build_system/config_windows.html#install-python-3

**Configuring the PATH environment variable:** https://aswf-openrv.readthedocs.io/en/latest/build_system/config_windows.html#path-environment-variable

**Comment by jaspreet-dimension** [comment]
I have just reinstalled Python and followed the steps you've posted but I'm still getting the same issue.

Running where python I can see the two Python installations - the docs say the top one will be used. I don't think this is the problem
```
$ where python
C:\Python311\python.exe
C:\msys64\mingw64\bin\python.exe
```

**Reply by bernie-laberge** [reply, marked as answer]
Hello @jaspreet-dimension ,
I believe I ran into the same issue recently and it had to do with the fact that I had a previous .venv directory created when I had the python from MSYS2 active.
My problem got solved after deleting the existing .venv directory
` rm -rf .venv` 
I hope this helps

**Reply by jaspreet-dimension** [reply]
Hey @bernie-laberge thanks, that was it!
